internal/resources/resource_host.go

```diff
@@ -194,6 +196,16 @@ func hostSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "Network ID of the default route",
 		},
+		hNetUntagged: {
+			Type:        schema.TypeString,
+			Description: "Untagged network",
+			Optional:    true,
+		},
```

Comment:
If this field cannot be changed later via Update, then ForceNew must be set to true.

Reply:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/65302544/189364981-691384e7-2e3b-40dc-996e-066771f465d2.png">
from API Docs, it is not an updatable field.

Reply:
Good catch I'll make a PR for that!